.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+## What is the purpose of this pull request?
+
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Typo fix
+- [ ] Other, please explain:
+
+## Check if your issue is already reported.
+
+If your pull request is a **Bug fix** or a **New feature** please look for the existing issues covering the same case.
+
+Have a look in [open issues list](https://github.com/ckeditor/ckeditor-dev/issues) using a search function. If your issue
+is already reported include the reference here.
+
+During our transition period you could also check [CKEditor's former bug tracker](https://dev.ckeditor.com). If the issue
+is there but not in GitHub, [create a new one](https://github.com/ckeditor/ckeditor-dev/issues/new) and put a reference link here.
+
+*Put all reference links here...*
+
+## Does your PR contains necessary tests?
+
+All patches which change the editor code must include tests. You can always read more
+on [PR testing](http://docs.ckeditor.com/#!/guide/dev_contributing_code-section-tests),
+[how to set the testing environment](http://docs.ckeditor.com/#!/guide/dev_tests) and
+[how to create tests](http://docs.ckeditor.com/#!/guide/dev_tests-section-creating-your-own-test)
+in the official CKEditor docs.
+
+Remember that you can always [propose an incomplete patch](http://docs.ckeditor.com/#!/guide/dev_contributing_code-section-proposing-incomplete-patches).
```

Comment:
Removed. It is still available in docs so if someone dives deeper the information will be still available.